build.gradle

```diff
@@ -24,6 +24,9 @@ ext {
           jacksonKotlin: 'com.fasterxml.jackson.module:jackson-module-kotlin:2.9.2',
           metricsParent: 'io.dropwizard.metrics:metrics-parent:3.1.0',
           metricsCore: 'io.dropwizard.metrics:metrics-core:3.1.0',
+          metricsGraphite: 'io.dropwizard.metrics:metrics-graphite:3.1.0',
+          metricsStackDriver: 'com.google.apis:google-api-services-monitoring:v3-rev425-1.23.0',
+          mockitoCore: "org.mockito:mockito-core:2.+",
```

Comment:
Is this actually being used? I don't see any `mockito` imports in the code.

Reply:
Removed. I was originally trying to mock stackdriver to test the actual batch sending code, but that turns out to be impossible to do a meaningful way given the structure of the client.